### ISSUE TITLE
Create an info page for DCC alternative sanctions

### ISSUE THREAD

**Issue by gabrielaperezdelaosa** [cityofaustin/techstack#4743]
> - [x] create a page that migrates https://www.austintexas.gov/department/alternative-sanctions using current content types
> - [x] make no additions to the text, you may delete redundant text
> - [ ] log content debt
>    - [x] note common term usage in the word bank 'Monstro'
>    - [x] create Github issues (tag as content debt)
>    - [x] also record content debt in Joplin notes
> content debt includes: fact checking, reading level, length, accessibility, questions about the user or the necessity of this page, user research debt, service design debt, policy lab work, and new features to dev.
**target date: 8/13**
> - [x] move to review and tag [reviewer]
>    - [ ] Reviewer proofreads the content
>    - [ ] If applicable, check implementation of fact checking
>    - [ ] Hit publish and copy URLs into the inventory
> If you can’t publish yet because you are waiting on the IA, you can still close the issue after the review is complete. 
**target date: 8/18**

**Comment by ablangworthy27** [comment]
Preview link: https://alpha.austin.gov/en/preview/information/UGFnZVJldmlzaW9uTm9kZToxNjk2 @srigdon 

Possible existing topics that this could live under: 
Parking, traffic and criminal violations

**Comment by srigdon** [comment]
Reviewed, edited, and added notes in Joplin. New preview page: https://alpha.austin.gov/en/preview/information/UGFnZVJldmlzaW9uTm9kZToxODA2 

Content debt:

1. Can this be made actionable and transformed into a service page? OR can it be shortened and folded into an existing service page?
2. The title and description refer to two alternative sanctions but there's nothing on the page about rehabilitation services, just community service. Edited the title and description for now, until we know more.
